uber/templates/summary/shirt_counts.html

```diff
@@ -9,7 +9,7 @@
 <h2>Shirt Counts</h2>
 
 {% for name, category in categories %}
-    <h3>{{ name }} shirts</h3>
+    <h3>{{ name }}</h3>
```

Comment:
All of the names already included the word "Shirt" so this just ended up saying "Staff Shirt shirts" and such.